.github/workflows/e2e-deploy.yml

```diff
@@ -24,105 +24,49 @@ on:
   pull_request:
     types:
       - "labeled"
+env:
```

Comment:
`e2e-deploy.yaml` is kind of different than what this workflow does, isn't it?
What this workflow actually does is a basic e2e testing, so just `e2e.yaml` seems good to me.

Reply:
Thank you for your suggestion 🙏
I have fixed it in the following commit.
https://github.com/vdaas/vald/pull/2141/commits/a56a631a807129f61d2a157a07274777833da73b